File: Capitulo_01/01_Getting_Started:_The_Easy_Problems/01_Super_Easy/12250_:::_Language_Detection/Main.java
import java.io.BufferedReader;
import java.io.File;
import java.io.FileReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main
{
	private static BufferedReader br;
	private static boolean test=true;

	public static void main(String[] args) throws IOException
	{
		if(test) br=new BufferedReader(new FileReader(new File("in.txt")));
		else br=new BufferedReader(new InputStreamReader(System.in));
		//-----------------------------------------------------------
		String linea, rta;
		for(int c=1; !(linea=br.readLine()).equals("#"); c++)
		{
			if(linea.contains("HELLO"))rta="ENGLISH";
			else if(linea.contains("HOLA"))rta="SPANISH";
			else if(linea.contains("HALLO"))rta="GERMAN";
			else if(linea.contains("BONJOUR"))rta="FRENCH";
			else if(linea.contains("CIAO"))rta="ITALIAN";
			else if(linea.contains("ZDRAVSTVUJTE"))rta="RUSSIAN";
			else rta="UNKNOWN";

			System.out.println("Case "+c+": "+rta);
		}
		//-----------------------------------------------------------
		br.close();
	}
}

File: Capitulo_01/01_Getting_Started:_The_Easy_Problems/01_Super_Easy/12250_:::_Language_Detection/Main.c
#include "stdio.h"
#include "string.h"

int test=1;

int main()
{
	if(test) freopen("in.txt","r",stdin);

	char s[20]; //14 + seguridad
	fgets(s, sizeof(s), stdin);

	int c;
	for(c=1; strcmp(s,"#")!=0; fgets(s, sizeof(s), stdin))
	{
		s[strlen(s)-1]='\0'; //fgets captures the enter
		
		printf("Case %d: ", c++);
		if(strcmp(s,"HELLO")==0) printf("ENGLISH\n");
		else if(strcmp(s,"HOLA")==0) printf("SPANISH\n");
		else if(strcmp(s,"HALLO")==0) printf("GERMAN\n");
		else if(strcmp(s,"BONJOUR")==0) printf("FRENCH\n");
		else if(strcmp(s,"CIAO")==0) printf("ITALIAN\n");
		else if(strcmp(s,"ZDRAVSTVUJTE")==0) printf("RUSSIAN\n");
		else printf("UNKNOWN\n");
	}

	return 0;
}
